lib/node-static.js

```diff
@@ -1,14 +1,16 @@
-'use strict';
-
-const fs     = require('fs')
-    , events = require('events')
-    // , buffer = require('buffer')
-    , http   = require('http')
-    , path   = require('path')
-    , mime   = require('mime')
-    , minimatch = require('minimatch')
-    , util   = require('./node-static/util')
-    , pkg = require('../package.json');
+import fs from 'fs';
+import events from 'events';
+import http from 'http';
+import path from 'path';
+// import buffer from 'buffer';
```

Comment:
what's with this here?

Reply:
That line had been based on a different commented out line (the same, but in CommonJS form). I can remove it if it wasn't there to remind you to do something :-)